apps/algorithm/search/src/server.py

```diff
@@ -223,21 +249,30 @@ async def search(
 
 
 @app.post("/api/search/reindex/listing-created")
-async def reindex_listing_created(listingId: str, authorization: str = Header(None)):
-    # actual logic will go here
+async def post_listing(listing: Listing, authorization: str = Header(None)):
```

Comment:
My assumption would be that fetching from the database would just be a better idea (since it's the source of truth).
```suggestion
async def post_listing(listingId: int, authorization: str = Header(None)):
```

Reply:
The Listing object defined here is different from the one defined in the database. We can update this in sprint 3 but for now it isn't broken so we're leaving it.

Reply:
Also this is following the definition laid out in the SRS